bindings/nodejs/lib/types/client/prepared-transaction-data.ts

```diff
@@ -23,7 +23,7 @@ export class PreparedTransactionData {
     /**
      * Optional remainder output information
      */
-    remainder?: Remainder;
+    remainders?: Remainder[];
```

Comment:
Not optional anymore?

Reply:
Still optional for bindings because of the way it's serialized (unless we don't want that)

Reply:
Ok